### PR TITLE
PR to unrevert "integrating updated bulk services/availability API""

### PR DESCRIPTION
**Pull request by mekarpeles**
Reverts ... revert  internetarchive/openlibrary#4213

### CODE REVIEW

File: openlibrary/core/lending.py [hunk 1]
```diff
@@ -137,8 +137,9 @@ def compose_ia_url(limit=None, page=1, subject=None, query=None, work_id=None,
     # If no lending restrictions (e.g. borrow, read) are imposed in
     # our query, we assume only borrowable books will be included in
     # results (not unrestricted/open books).
-    if (not query) or ('loans__status__status:' not in query):
-        q += ' AND loans__status__status:AVAILABLE'
+    lendable = '(lending___available_to_browse:true OR lending___available_to_borrow:true)'
```
Comment (cclauss) [inline openlibrary/core/lending.py:140]:
```suggestion
    lendable = (
        '(lending___available_to_browse:true OR lending___available_to_borrow:true)'
    )
```

File: openlibrary/plugins/openlibrary/home.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 CAROUSELS_PRESETS = {
     'preset:thrillers': '(creator:"Clancy, Tom" OR creator:"King, Stephen" OR creator:"Clive Cussler" OR creator:("Cussler, Clive") OR creator:("Dean Koontz") OR creator:("Koontz, Dean") OR creator:("Higgins, Jack")) AND !publisher:"Pleasantville, N.Y. : Reader\'s Digest Association" AND languageSorter:"English"',
     'preset:comics': '(subject:"comics" OR creator:("Gary Larson") OR creator:("Larson, Gary") OR creator:("Charles M Schulz") OR creator:("Schulz, Charles M") OR creator:("Jim Davis") OR creator:("Davis, Jim") OR creator:("Bill Watterson") OR creator:("Watterson, Bill") OR creator:("Lee, Stan"))',
-    'preset:authorsalliance_mitpress': '(openlibrary_subject:(authorsalliance) OR collection:(mitpress) OR publisher:(MIT Press) OR openlibrary_subject:(mitpress)) AND (!loans__status__status:UNAVAILABLE)'
+    'preset:authorsalliance_mitpress': '(openlibrary_subject:(authorsalliance) OR collection:(mitpress) OR publisher:(MIT Press) OR openlibrary_subject:(mitpress))'
```
Comment (cclauss) [inline openlibrary/plugins/openlibrary/home.py:28]:
```suggestion
    'preset:authorsalliance_mitpress': (
        '(openlibrary_subject:(authorsalliance) OR collection:(mitpress) OR '
        'publisher:(MIT Press) OR openlibrary_subject:(mitpress))'
    )
```

**Review by cclauss — CHANGES_REQUESTED** [review]
Wrap long lines.

**Review by cclauss — APPROVED** [review]
RSLGTM

**Review by cdrini — APPROVED** [review]
Lgtm!

- ✅ Tested: http://staging.openlibrary.org/subjects/literature
    - ✅ Second page loads
    - ❌ Second page availability correct; Looks like the JS might have its own logic for checking this, since it has its own logic for rendering stuff here :( Not a blocker on this PR though.
- http://staging.openlibrary.org/books/OL18336664M/Harry_Potter_and_the_Prisoner_of_Azkaban
    - ✅ Shows available in sidebar
    - ✅ Shows available in first row of table
    - ✅ After checked out, shows "Read" in sidebar
    - ✅ After checked out, incognito shows "Join Waitlist"
- http://staging.openlibrary.org/authors/OL6951711A/Tom_Clancy
- http://staging.openlibrary.org/works/OL449016W/The_hunt_for_Red_October?edition=huntforredoctobe1990clan Not in Library

**Pull request merged**